### DISCUSSION TITLE
Issues with wyze app and live view when wyze bridge is running

### DISCUSSION THREAD

**Discussion by drkavnger99** [mrlt8/docker-wyze-bridge#758] [Q&A]
Not sure if anyone has seen this.  The Wyze app connection to the cameras as well as the Wyze webview is basically broken when wyze-bridge is running.  Also connection to the cameras goes up and down with wyze-bridge.  This started happening after latest update.

Current setup 13 cameras total across 2 different locations (8 I own and 5 shared with me).
At my home I have 
3 x Wyze Cam v2's with RTSP firmware
1 x Wyze Cam v2 with latest wyze official firmware
1 x OG Telephoto
1 x OG 
2 x Wyze Cam Pan with RTSP firmware

At my buddy's place all Wyze firmware
4 x Wyze Cam v3's
1 x Wyze Cam Outdoor

When wyze-bridge is running it seems it can't restore connection to any of the cameras for more than a few seconds and then it's up and down from that point on.  Stopping the docker image the apps all work fine no drops.

Going to try and downgrade to previous version to see if there is a difference.

**Comment by mrlt8** [comment]
Web view is using a different protocol so it shouldn't be affected by the bridge. Could it be a multicasting issue? 

Could you try limiting the bridge to a single camera with `- FILTER_NAMES=cam name`  to see if that still breaks the web view and app?

**Reply by drkavnger99** [reply]
Spent some time going through the issues.  For clarity sakes here's how I'm set up.
Wyze-Bridge on docker on UNRAID
Frigate running on HA with coral M.2 card

When I run wyze-bridge with audio enabled and HD60 for all cameras I start losing ping to the camera's.  I've gone through and disabled onboard sd-card recording and all the AI Detect/motion tagging etc which made no difference.  All but one camera I'm connecting to is running the RTSP wyze firmware.  Wifi is Ubiquiti AC Pro's with no signal issues to the devices.
Config options:
FRESH_DATA:FALSE
FORCE_ENCODE:FALSE
RTSP_FW:FALSE
SUBSTREAM:FALSE
ENABLE_AUDIO:FALSE
SNAPSHOT:RTSP60
QUALITY:SD30
FILTER_NAMES:new OG cams and another sites cameras
FILTER_BLOACK:TRUE
SUB_QUALITY:SD30

If I go to HD60 I'll get intermittent drops of the cameras and HD120 forget it.  These drop from 2-3MS ping to 1000+ then not connected.  App doesn't work with them in this state and neither does wyze-bridge.  Below is what my two worst performing Wyze V2 Cams that seem to do it a bit more often than the rest.

No Audio Enabled and SD30
[front-yard] 📡 Getting 30kb/s SD stream (h264/15fps) via LAN mode (WiFi: 96%) FW: 4.28.4.40 (2/3)
[garage] 📡 Getting 30kb/s SD stream (h264/15fps) via LAN mode (WiFi: 98%) FW: 4.28.4.41 (2/3)

Enabled Audio and HD60

[front-yard] 📡 Getting 30kb/s SD stream (h264/15fps) via LAN mode (WiFi: 98%) FW: 4.28.4.40 (2/3)
[front-yard] 🔊 Audio Enabled - PCM > LIBOPUS/8,000Hz
[garage] 📡 Getting 30kb/s SD stream (h264/15fps) via LAN mode (WiFi: 94%) FW: 4.28.4.41 (2/3)
[garage] 🔊 Audio Enabled - PCM > LIBOPUS/8,000Hz

Reply from 192.168.100.10: bytes=32 time=3ms TTL=63
Reply from 192.168.100.10: bytes=32 time=3ms TTL=63
Reply from 192.168.100.10: bytes=32 time=14ms TTL=63
Reply from 192.168.100.10: bytes=32 time=1ms TTL=63
Reply from 192.168.100.10: bytes=32 time=69ms TTL=63
Reply from 192.168.100.10: bytes=32 time=40ms TTL=63
Reply from 192.168.100.10: bytes=32 time=10ms TTL=63
Reply from 192.168.100.10: bytes=32 time=36ms TTL=63
Reply from 192.168.100.10: bytes=32 time=12ms TTL=63
Reply from 192.168.100.10: bytes=32 time=1102ms TTL=63
Reply from 192.168.100.10: bytes=32 time=342ms TTL=63
Reply from 192.168.100.10: bytes=32 time=1624ms TTL=63
Reply from 192.168.100.10: bytes=32 time=1162ms TTL=63
Reply from 192.168.100.10: bytes=32 time=757ms TTL=63
Reply from 192.168.100.10: bytes=32 time=1790ms TTL=63
Reply from 192.168.100.10: bytes=32 time=1960ms TTL=63
Reply from 192.168.100.10: bytes=32 time=1000ms TTL=63
Reply from 192.168.100.10: bytes=32 time=814ms TTL=63
Reply from 192.168.100.10: bytes=32 time=1576ms TTL=63
Reply from 192.168.100.10: bytes=32 time=898ms TTL=63
Reply from 192.168.100.10: bytes=32 time=2308ms TTL=63
Reply from 192.168.100.10: bytes=32 time=912ms TTL=63
Reply from 192.168.100.10: bytes=32 time=294ms TTL=63
Reply from 192.168.100.10: bytes=32 time=884ms TTL=63
Reply from 192.168.100.10: bytes=32 time=847ms TTL=63
Reply from 192.168.100.10: bytes=32 time=122ms TTL=63
Reply from 192.168.100.10: bytes=32 time=1637ms TTL=63
Reply from 192.168.100.10: bytes=32 time=2396ms TTL=63
Request timed out.
Request timed out.

Is this to be expected due to the added load or should this be able to do it?  The ping flaps back and forth. it will work for a bit (seconds/minutes) then start doing it again.

**Reply by mrlt8** [reply]
> Wifi is Ubiquiti AC Pro

This is a known issue with Ubiquiti gear. I believe it has something to do with the way the tutk SDK uses multicast/IGMP that cause the 2.4Ghz to channel to get flooded and bring down all the devices.

**Comment by jvanderaa** [comment]
I have a similar experience here on this one. I've got 5 outdoor cams - V2, split across two bridges. I'm working on trying to get the Wyze Cams into HomeAssistant. The times that I get one of the cameras to be reading by Wyze-Bridge, the cameras in the app go offline. The fix is to reboot the outdoor Wyze Outdoor-bridge (not the app). The Wyze Outdoor-bridge is Ethernet connected. But I do have all Uniquiti gear, which as mentioned earlier had issues (though that being the WiFi side, maybe, just maybe Ubiquiti's LAN product would be OK.....).

**Reply by mrlt8** [reply]
See this discussion re: unifi https://github.com/mrlt8/docker-wyze-bridge/discussions/891
